recipe/meta.yaml

```diff
@@ -29,11 +29,12 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - libffi {{ libffi }} # [not win]
```

Comment:
worth running the tests? I can't see any compatibility description upstream with libffi. If this is urgent, I won't block (so approved)

Reply:
I think the static linker performs symbol resolution for shared object dependencies during compile time, so the symbols it needs should be present, at least. And we've got the latest maj.min version of libcffi. so should be fine I think